src/icons/StaticArrow.tsx

```diff
@@ -0,0 +1,53 @@
+export default function ({ turnUp }: { turnUp?: boolean }) {
```

Comment:
You can do `<Arrow disabled />` and for open-closed state use `<Arrow disabled turnDown />`